easy/Challange5.js

```diff
@@ -0,0 +1,20 @@
+//Write a function that takes a two-digit number and determines if it's the largest of two possible digit swaps.
+
+//example. largestSwap(27) ➞ false   largestSwap(43) ➞ true
+
+function largestSwap(num)
+{
+    let initial=num;
+    let temp=num;
+    num=num%10;
```

Comment:
Never mutate (Change) values received in the function parameters.
In this case `num` is received as function parameter, so never change its value.
Go through this article : https://www.freecodecamp.org/news/what-is-a-pure-function-in-javascript-acb887375dfe/

Reply:
replaced the variable num with initial which is created temporary for performing actions
commit ID ddc0478990d53f84dcf0ffe1f440d3a1b898c356